src/az_ulib_ipc/az_ulib_ipc.c

```diff
@@ -549,53 +697,98 @@ static az_result report_interfaces(uint16_t start, az_span* result, uint16_t* ne
   for (interface_index = start; interface_index < AZ_ULIB_CONFIG_MAX_IPC_INTERFACE;
        interface_index++)
   {
-    if (_az_ipc_cb->_internal.interface_list[interface_index].interface_descriptor != NULL)
+    const volatile az_ulib_interface_descriptor* const descriptor
+        = _az_ipc_cb->_internal.interface_list[interface_index].interface_descriptor;
+    if (descriptor != NULL)
     {
-      int32_t next_size = az_span_size(_az_ipc_cb->_internal.interface_list[interface_index]
-                                           .interface_descriptor->_internal.name);
-      char version_str[12];
-      az_span version_span = AZ_SPAN_FROM_BUFFER(version_str);
+      int32_t next_size = az_span_size(descriptor->_internal.pkg_name)
+          + az_span_size(descriptor->_internal.intf_name)
+          + 6; // 6 = '"', '*', '.', '.', '.' and '"'
```

Comment:
>6; // 6 = '"', '*', '.', '.', '.' and '"' [](http://example.com/codeflow?start=12&length=41)

thoughts on making this a #define?

Reply:
In this case, I think the explained number is better than a constant.